pixeldata/src/encapsulation.rs

```diff
@@ -0,0 +1,80 @@
+//! DICOM Pixel encapsulation
+//!
+//! This module implements encapsulation for pixel data.
+use dicom_core::value::fragments::Fragments;
+use dicom_core::value::Value;
+
+/// Encapsulate the pixel data of the frames. If fragment_size > 0 it will use 1 fragment per frame.
+/// This parameter is ignored for multi frame data, as 1 fragment per frame is required.

```

Comment:
isn't that depended on the transfer syntax used?
https://dicom.nema.org/dicom/2013/output/chtml/part05/sect_A.4.html

Reply:
I think it is better to have 1 function for this, otherwise when you intend to encapsulate you need to verify if it is single or multi frame, then call the corresponding function.